src/lava/proc/conv/models.py

```diff
@@ -27,14 +27,24 @@ class AbstractPyConvModel(PyLoihiProcessModel):
     dilation: np.ndarray = LavaPyType(np.ndarray, np.int8, precision=8)
     groups: np.ndarray = LavaPyType(np.ndarray, np.int8, precision=8)
 
+    def __init__(self) -> None:
+        super().__init__()
+        self.a_buf = None
+
     def run_spk(self) -> None:
         s_in = self.s_in.recv()
         a_out = utils.conv(
             s_in, self.weight,
             self.kernel_size, self.stride, self.padding, self.dilation,
             self.groups[0]
         )
-        self.a_out.send(self.clamp_precision(a_out))
+
+        if self.a_buf is None:
+            self.a_buf = np.zeros_like(a_out)
+
+        # self.a_out.send(self.clamp_precision(a_out))
```

Comment:
If we do not need this line of code anymore, we should delete it, instead of commenting it out